### PR TITLE
Allow to define radio button form via settings

### PR DESCRIPTION
**Pull request by berinhard**
This was required because I needed to style how the form are being rendered in PyCon 2021 website.

### CODE REVIEW

File: registrasion/forms.py [hunk 1]
```diff
@@ -1319,11 +1320,14 @@ def contains_errors(self):
         required=False,
     )
 
-try:
-    VoucherForm = util.get_object_from_name(settings.VOUCHER_FORM)
-except:
-    VoucherForm = _VoucherForm
+def get_form_class_from_settings(config_name, default):
```
Comment (ewdurbin) [inline registrasion/forms.py]: might suggest moving this up to the top of the file for discoverability.

File: registrasion/forms.py [hunk 2]
```diff
@@ -168,7 +168,7 @@ def ProductsForm(category, products, sold_out_products=None, disabled_products=N
     cat = inventory.Category
     RENDER_TYPES = {
         cat.RENDER_TYPE_QUANTITY: _QuantityBoxProductsForm,
-        cat.RENDER_TYPE_RADIO: _RadioButtonProductsForm,
+        cat.RENDER_TYPE_RADIO: get_form_class_from_settings('RADIO_BUTTON_PRODUCTS_FORM', _RadioButtonProductsForm),
```
Comment (ewdurbin) [inline registrasion/forms.py]: two things:

  - can we prefix these settings with `REGISTRASION_`
  - should we apply the same logic to all render types while we're here?

Comment (berinhard) [reply registrasion/forms.py]: Agree with both!

File: registrasion/forms.py (hunk 1 above)
Comment (berinhard) [reply registrasion/forms.py]: Fixed in da25274b34fb2fd0f26bc0f1e44961ac6bf0fc38

File: registrasion/forms.py (hunk 2 above)
Comment (berinhard) [reply registrasion/forms.py]: Fixed in:

- 9bb88d5c1c61067295b9753aac182290db87e1d2
- 1fd10a71f04ee3a36013811bbb26321e52e02f02

**Review by ewdurbin — APPROVED** [review]

**Comment by ewdurbin** [comment]
Looks great @berinhard! We probably want a matching PR to update the Attendee Profile and Voucher form setting names in the pycon2021 codebase to merge with this one, right?

**Comment by berinhard** [comment]
Sure, here you are: https://github.com/psf/pycon2021/pull/13/commits/4c1123d20043b009c80c0c82ecb780eac34a587a

It's the most recent one in the [PR #13](https://github.com/psf/pycon2021/pull/13/).

**Comment by ewdurbin** [comment]
@berinhard can we break that out separately to merge this first?

**Comment by berinhard** [comment]
Not sure if I followed you... Should I split new settings variables in 2 PRs, one for registrasion and other for PyCon?

**Comment by ewdurbin** [comment]
just a PR for the commit referenced: https://github.com/psf/pycon2021/commit/4c1123d20043b009c80c0c82ecb780eac34a587a

That will allow us to merge this and that PR simultaneously then the rest of https://github.com/psf/pycon2021/pull/13/ later after more thorough review.

**Comment by berinhard** [comment]
Here you are @ewdurbin:

https://github.com/psf/pycon2021/pull/14

**Comment by ewdurbin** [comment]
perfect. thank you!

**Pull request merged**